### PR TITLE
Make the bulk output slimmer

### PR DESCRIPTION
**Pull request by guenhter**
Combine two output lines of the git-bulk command into one for the non-guarded mode.

### CODE REVIEW

File: bin/git-bulk [hunk 1]
```diff
@@ -141,7 +142,6 @@ function executBulkOp () {
       eval cd "\"$gitrepodir\"" # into git repo location
       local curdir=$(pwd)
       local leadingpath=${curdir#${actual}}
-      [ "${quiet?}" != "true" ] && echo 1>&2 "Current repository: ${leadingpath%/*}/${bldred}${curdir##*/}${reset}"
```
Comment (spacewander) [inline bin/git-bulk:144]: Why remove this prompt?

Comment (guenhter) [reply bin/git-bulk:144]: It is included now in the the line where it prints: "Executing git something". Now it it

"Executing git something in repository yourrepo"

Comment (spacewander) [reply bin/git-bulk:144]: I see

**Comment by spacewander** [comment]
Let's merge master to solve the conflict.

**Review by spacewander — APPROVED** [review]

**Pull request merged**